tools/docker/builder/openwrt/Dockerfile

```diff
@@ -44,12 +44,18 @@ RUN git clone "$OPENWRT_REPOSITORY" openwrt \
 
 WORKDIR /home/openwrt/openwrt
 
+COPY --chown=openwrt:openwrt configs/* /home/openwrt/
+
 RUN if [ -n "$INTEL_FEED" ] ; then \
 	echo "src-git intel $INTEL_FEED" >> feeds.conf.default \
 	&& scripts/feeds update intel \
 	&& scripts/feeds install intel_mips ; \
+    fi ;
+    
+RUN if [ -e /home/openwrt/${TARGET}_config ]; then \
+        cp /home/openwrt/${TARGET}_config .config ;\
```

Comment:
After thinking about it, I think it might be more useful to split it between different SDK configuration rather than between targets.
What I mean is that we could have a file say sdk_default which would be the configuration we used until now, and another one called sdk_gcc8 for example, so that any target could easily be switched between different GCC versions (it could even be integrated later as an option in the builds script, although that's not mandatory).

If you anyway want to go for the per-target approach, you still need an argument to decide which file to copy, so you could remove the if statement entirely (and copy the corresponding config file in any case).

Reply:
Done.